src/config.ts

```diff
@@ -432,7 +432,10 @@ export const GAS_SCHEDULE_V1: FeeSchedule = {
             GAS_SCHEDULE_V1[firstHop.source] === undefined ? 0 : GAS_SCHEDULE_V1[firstHop.source](firstHop.fillData);
         const secondHopGas =
             GAS_SCHEDULE_V1[secondHop.source] === undefined ? 0 : GAS_SCHEDULE_V1[secondHop.source](secondHop.fillData);
-        return new BigNumber(firstHopGas).plus(secondHopGas).toNumber();
+        return new BigNumber(firstHopGas)
+            .plus(secondHopGas)
+            .plus(30e3)
```

Comment:
adding 30k here to account for the fact that MultiHop incurs the overhead of the FQT twice, whereas the `exchangeProxyOverhead` only accounts for one FQT call.